app/models/course.rb

```diff
@@ -323,15 +314,69 @@ class Course < ActiveHash::Base
             'You are an intermediate-level developer',
             'You might be familiar with version control and basic web architecture'
           ]
+        }
+      ]
+    }, {
+      id: 8,
```

Comment:
It's no worries @anymoto, there'd be no wait for you to know there is a course for it unless I told you! Otherwise it indeed looks like it's just a translation of the course description. ;)